tools/taskcluster/accept-license.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# If a command fails, do not fail the script
+set -ex
+# Accept SDK license
+if [[ $(type -P "sdkmanager") ]]; then
+    yes | sdkmanager "platforms;android-28" "build-tools;28.0.3"
+elif [[ $(type -P "android") ]]; then
+    (while sleep 3; do echo "y"; done) | android update sdk --no-ui --all --filter android-28
```

Comment:
So this was focus code but `set -e` allows pipeline exit (`android update sdk --no-ui --all --filter android-28` or `"platforms;android-28" "build-tools;28.0.3"`) IFF its return with error code. We want to make sure the pipe commands run in case CI servers have latency issues but I guess we could implement a retry counter here. What do you think, @mcomella ?

Reply:
I'm not quite sure I understand but I don't know that this script needs to exist (see above).